tokio-console/src/view/task.rs

```diff
@@ -83,7 +94,7 @@ impl TaskView {
                         // controls
                         layout::Constraint::Length(controls.height()),
                         // task stats
-                        layout::Constraint::Length(10),
+                        layout::Constraint::Length((10 + location_lines_vector.len() - 1) as u16),
```

Comment:
And use that variable here:

```suggestion
                        layout::Constraint::Length(task_stats_height),
```